src/parser/transforms.js

```diff
@@ -268,6 +269,10 @@ module.exports.preprocessGlimmerTemplates = function preprocessGlimmerTemplates(
         for (const part of n.tag.split('.')) {
           const regex = new RegExp(`\\b${part}\\b`);
           const match = codeSlice.match(regex);
+
+          // named-blocks are not ElementNodes
+          if (!match && part.startsWith(':')) continue;
```

Comment:
should still split it into parts, even if there is only one. if its references for an eslint error

Reply:
there can't be eslint errors with named blocks tho?

Reply:
maybe not, or maybe not yet :)

Reply:
Happy to have this code undone -- we just gotta get some examples in the repo.
All the tests in eslint-plugin-ember were using components, it turns out -- we had no coverage for elements, named-blocks, etc!!

Reply:
hmm,
it should have made a difference here:
https://github.com/ember-cli/eslint-plugin-ember/blob/master/tests/lib/rules-preprocessor/gjs-gts-parser-test.js#L262
ah, but you actually didn't change it for elements, only for named-blocks.

Reply:
lets also add parts for named-blocks to keep it consistent. then we don't need to changes in the other code parts